.github/workflows/ci.yml

```
@@ -187,9 +187,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
 
       - name: Check whether this phase should run
-        run: echo "GIT_DIFF=$( git diff --name-only HEAD^ | egrep -ic 'crossdc|infinispan' )" >> $GITHUB_ENV
+        run: echo "GIT_DIFF=$(( $( git diff --name-only HEAD^ | egrep -ic 'crossdc|infinispan' ) | $( git diff HEAD^ pom.xml | egrep -ic '\+\s+<wildfly.version>' ) ))" >> $GITHUB_ENV
```

Did you mean the following?
```suggestion
        run: echo "GIT_DIFF=$[ $( git diff --name-only HEAD^ | egrep -ic 'crossdc|infinispan' ) + $( git diff HEAD^ pom.xml | egrep -ic '\+\s+<wildfly.version>' ) ]" >> $GITHUB_ENV
```

---

Yeah, that would work too to get `GIT_DIFF` to evaluate to non-zero if Wildfly version got bumped. And is actually more readable due not needing to add that much brackets.

@hmlnarik Thanks for the suggestion! Added it into the 1-th commit & squashed.